### ISSUE TITLE
Findable Help: Some suggested editorial changes for clarity

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#1462]
Suggested edits include:

- Not sure why the scoping is needed. If there's help, make it appear in the same relative order. There could be a note to clarify, if needed.
- Should use the word "help" in the SC text. Currently it is only in the short name for the SC.
- The text in green block should replace the quotation after "information" with a period. Couldn't tell if this is planned to be part of the SC or not, as in other places those green blocks were used to notate Editor's questions for the draft review.

With the edits incorporated, the SC would read: 

_For any set of Web pages, if any of the following forms of help are available, then at least one is included on each page in the same position relative to other content._

_Note: A single-page Web application that provides multiple pages from a single URI is considered a set of Web pages._

**Comment by mraccess77** [comment]
I suspect order may have been originally used because it implies a programmatic order in the DOM for assistive technology compared to position -- which tends to be a visual thing but not necessarily corresponding in the reading order.

**Comment by maryjom** [comment]
@mraccess77 Yes, another thing that made me pause. Which order is this referring to?  Some persons with cognitive disabilities have content read to them so which order is this SC scoped to - visual or reading?

**Comment by alastc** [comment]
Draft response:
<hr>

Looking at the proposed text:

> For any set of Web pages, if any of the following forms of help are available, then at least one is included on each page in the same position relative to other content.

And comparing to the [current version](https://w3c.github.io/wcag/guidelines/22/#findable-help ): 
> For any set of Web pages, if one or more of the following ways of finding help is supported, then access to at least one way of finding help is included in the same relative order on each page:

We have a few differences which are already things we've been over:
-	SPA need something to cover the one-URL style SPAs, see the discussion here: https://github.com/w3c/wcag/issues/1347#issuecomment-740211560 
-	"One or more" vs "any of the following" – could go either way, doesn't seem to make a huge difference.
-	Using "ways of finding help" – indicates it is on the page, whereas the intent is that it could be links through to other pages.
-	“is supported” – we had a big discussion on available vs supported, went with supported.
-	“access to” vs "is included", again it is important to indicate the help itself may not be on the page.

Although the group would like to simplify the text whenever possible, there are reasons for the current text that are important to have the correct scope. If the group agrees with this response we will close the issue, but please re-open if you feel one of those decisions should be re-evaluated, with reasoning.

**Comment by alastc** [comment]
The draft response above accepted by the group.
